packages/cozy-mespapiers-lib/package.json

```diff
@@ -45,6 +48,12 @@
     "cozy-realtime": ">=4.2.0",
     "cozy-sharing": ">=4.3.0",
     "cozy-ui": ">=70.5.1",
-    "react-router-dom": ">=5.2.0"
+    "react-router-dom": ">=5.2.0",
+    "react-dom": "18.0.0",
+    "react": "18.0.0"
+  },
+  "resolutions": {
+    "cozy-authentication/react-dom": "18.0.0",
```

Comment:
Can you explain the need of this line? mespapiers-lib doesn't depend on cozy-authentication so it seems weird to me

Reply:
This PR is a draft containing all my effort in order to get react aand react-dom being used with 18.0.0 version. It hasn't been cleant yet. I will have a llok later.